docs/traceflow-guide.md

```diff
@@ -27,24 +31,27 @@ You need to switch on traceflow from featureGates defined in antrea.yml for both
     featureGates:
     # Enable traceflow which provides packet tracing feature to diagnose network issue.
       Traceflow: true
+    # Enable antrea proxy which provides ServiceLB for in-cluster services in antrea agent.
```

Comment:
I saw we use both "Antrea" and "antrea". I thought it should be "Antrea".

Reply:
services -> Services

Reply:
Will make the change. This is taken straight from the antrea-agent.conf file, so I'll update it everywhere.

Reply:
Done

Reply:
Done, thanks